tools/topology/topology2/include/pipelines/cavs/asrc-dai-copier-playback.conf

```diff
@@ -0,0 +1,104 @@
+#
+# CAVS ASRC capture pipeline
+#
+# A simple asrc pipeline. All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.asrc-dai-copier-playback.N.attribute_name".
+#
+# Usage: asrc-dai-copier-playback pipeline object can be instantiated as:
+#
+# Object.Pipeline.asrc-dai-copier-playback."N" {
+# 	format		"s16le"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/dai-copier.conf>
+<include/components/asrc.conf>
+<include/components/pipeline.conf>
+
+Class.Pipeline."asrc-dai-copier-playback" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		unique	"instance"
+	}
+
+	Object.Widget {
+		dai-copier."1" {
+			type dai_in
+			node_type $I2S_LINK_OUTPUT_CLASS
+			num_input_audio_formats 1
+			num_output_audio_formats 1
+			num_input_pins 1
+
+			Object.Base.audio_format [
+				{
+					in_bit_depth		32
+					in_valid_bit_depth	32
+					out_bit_depth		32
+					out_valid_bit_depth	32
+				}
+			]
+		}
+
+		asrc."1" {
+			asynchronous_mode	1
+			operation_mode		0
+			rate_in 48000
+			num_input_audio_formats 3
+			Object.Base.input_audio_format [
+				{
+					in_bit_depth            16
+					in_valid_bit_depth      16
+				}
+				{
+					in_bit_depth            32
+					in_valid_bit_depth      24
+				}
+				{
+					in_bit_depth            32
+					in_valid_bit_depth      32
+				}
+			]
+			num_output_audio_formats 1
+			Object.Base.output_audio_format [
+				{
+					out_bit_depth           32
+					out_valid_bit_depth     32
+				}
+			]
```

Comment:
Why 3 input audio formats and one output? The asrc is not supporting format conversion, but same input and output format for s16/s24/s32.